Configuration/StandardSequences/python/DigiToRaw_cff.py

```diff
@@ -38,17 +38,20 @@
 phase2_tracker.toReplaceWith(DigiToRaw, DigiToRaw.copyAndExclude([siPixelRawData])) # FIXME
 
 # GEM settings
-_run2_GEM_2017_DigiToRaw = DigiToRaw.copy()
-_run2_GEM_2017_DigiToRaw.insert(-1,gempacker)
+_gem_DigiToRaw = DigiToRaw.copy()
+_gem_DigiToRaw.insert(-1,gempacker)
 
-_run3_DigiToRaw = _run2_GEM_2017_DigiToRaw.copy()
-_run3_DigiToRaw.insert(-1,me0packer)
+_me0_DigiToRaw = _gem_DigiToRaw.copy()
```

Comment:
(As above) it seems safer to me not to make the different settings depend on each other.
Therefore i would copy() here DigiToRaw once again, instead of an already modified version of it, and insert() the gempacker and me0packer on it